File: src/main/java/com/train/Tester.java
package com.train;

import java.util.Scanner;

public class Tester {
    public static void main(String[] args) {
        System.out.print("Please enter number of tickets: ");
        Scanner scanner = new Scanner(System.in);
                int number = scanner.nextInt();
        System.out.print("How many round-trip tickets: ");
                int roundtrip = scanner.nextInt();
        Ticket ticket = new Ticket(number, roundtrip);
        ticket.print();
    }
}

File: src/main/kotlin/com/train/Tester.kt
package com.train

import java.util.*

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    println("Please enter number of tickets: ")
        var number = scanner.nextInt()
    println("How many round-trip tickets: ")
        var roundtrip = scanner.nextInt()
    val ticket = Ticket(number,roundtrip)
    ticket.print()
}

class Ticket(var number:Int, var roundtrip: Int){
    fun print(){
        println(((number-roundtrip)*1000)+(roundtrip*2000*0.9))
    }
}
